backend_nestjs/dining-service/src/table-orders/controllers/table-orders.controller.ts

```diff
@@ -59,6 +59,7 @@ export class TableOrdersController {
   @ApiUnprocessableEntityResponse({ type: TableOrderAlreadyBilledException, description: 'TableOrder is already billed' })
   @Post(':tableOrderId')
   async addMenuItemToTableOrder(@Param() getTableOrderParams: GetTableOrderParams, @Body() addMenuItemDto: AddMenuItemDto): Promise<TableOrder> {
+    console.log('passing by here iam a controller');
```

Comment:
Can you delete this log? From my point of view, it does not contribute anything for execution or debugging.